client/store.js

```diff
@@ -9,7 +9,29 @@ const podcasts = function podcastList(state = [], action) {
   }
 }
 
-const reducer = Redux.combineReducers({ podcasts })
+const searchInput = function newSearch(state = '', action) {
+  switch(action.type) {
+    case 'INPUT_CHANGED':
+      return action.text
+    case 'SEARCH_CREATED':
+      return ''
+    default:
+      return state
+  }
+}
+
+const searchPodcast = function searchedPodcast(state = [], action) {
+  switch(action.type) {
+    case 'SEARCH_LOADED':
+      return [...state, action.searchedPodcast]
```

Comment:
every time the search results come back, I think we want to replace the new state, and not append with the old state

```js
  return [...action.searchedPodcast]
```